### ISSUE TITLE
Suggested resources page

### ISSUE THREAD

**Issue by egiurleo** [wnbrb/wnb-rb-site#53]
## Problem

Our community frequently recommends resources (articles, podcasts, books, etc.) to each other, but we currently have no place to capture all those recommendations.

## Solution

There should be a page on the website where community members can suggest resources. That way, anybody looking for suggestions can easily draw upon the collective knowledge of the group without having to ask the same question again.

## Notes

This is in the "idea" phase, so anybody who takes on this task should suggest their preferred solution before starting to code.

---
**Current Status:**
This is a proposed feature that has been in the idea stage for 3 years. While it’s not a critical feature, it should be discussed with the website team to determine whether it should remain open.

**Comment by jrussell416** [comment]
**Issue Review**

**Recommendation:** Discuss Further
**Reason:** This issue was added to the repo while still in the idea phase and has remained untouched for three years. Further discussion is needed to decide whether to move forward with this idea and, if so, how to proceed.
 Note we already have Slack channels where resources are shared.
 
**Related Pull Request:**N/A

**Next Steps:** Discuss with issue author?

**Comment by sarah-fflow** [comment]
Suggest we keep resources in slack (or discord), to encourage membership.

**Comment by sarah-fflow** [comment]
We'd still like a resources page, even though yes, we can now keep resources in discord.

We will crowd source ideas from the community, but also, it would be nice to have a list of projects from our own community. Here's a start:

StoryGraph
https://www.simplefileupload.com/
https://fflow.io/
https://www.theodinproject.com/

@jrussell416 I think you were interested in this one?

**Comment by sarah-fflow** [comment]
Fixed under #436
